Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/messaging/builder/AdhocQueryRequestBuilder.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2009-2015, United States Government, as represented by the Secretary of Health and Human Services.  * All rights reserved. * All rights reserved.
+ *
```

Comment:
Why are we putting these classes in CONNECTCore?  Wouldn't it be better to put them in Document Query Core.  Connect core should be for across service functionality.  All of the following classes are specific to DQ.